py4DSTEM/process/diskdetection/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .probe import *
 from .diskdetection import *
+from .diskdetection_aiml import *
```

Comment:
As is, this should result in throwing errors for all normal installations, because importing py4DSTEM will import diskdetection_aiml which will try to import both tensorflow and crystal4D, which will not be installed for normal installations.  I suggest removing this import.  To make use of the AIML code then, users would need to (1) run the AIML installation, and then (2) in their scripts/notebooks/whatever, first import py4DSTEM, then import this specific module.

Reply:
or - change to a try statement

Reply:
or - import inside some relevant fn(s)